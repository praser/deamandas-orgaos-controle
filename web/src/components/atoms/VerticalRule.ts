import styled from 'styled-components';

const VerticalRule = styled.div`
  /* Layout */
  box-sizing: border-box;
  display: block;
  height: 38px;
  margin-bottom: 16px;
  margin-left: 16px;
  margin-right: 16px;
  margin-top: 16px;
  width: 1px;

  /* Appearance */
  border-right-color: rgb(227, 230, 240);
  border-right-style: solid;
  border-right-width: 1px;
`;

export default VerticalRule;
