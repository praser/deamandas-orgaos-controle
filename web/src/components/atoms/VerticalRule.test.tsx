import React from 'react';
import { render, screen } from '@testing-library/react';
import VerticalRule from './verticalRule';

describe('VerticalRule atomic component', () => {
  let vRule: HTMLElement;

  beforeEach(() => {
    render(<VerticalRule data-testid="v-rule" />);
    vRule = screen.getByTestId('v-rule');
  });

  it('is expected to match snapshot', () => {
    expect(vRule).toMatchSnapshot();
  });
});
