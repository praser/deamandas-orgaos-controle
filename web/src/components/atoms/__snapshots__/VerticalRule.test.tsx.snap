// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VerticalRule atomic component is expected to match snapshot 1`] = `
<div
  class="sc-bdvvaa beNmra"
  data-testid="v-rule"
/>
`;
